src/components/Create.js

```diff
@@ -30,95 +30,229 @@ import {
   TEXT_INPUT,
   SLIDER,
   QUESTION_TYPE,
+  QUESTION_LIST_TYPE,
+  NEW_QUESTION_TYPE,
 } from "./constants";
 import { useAppData } from "./context/hooks";
+import { getDataWithId, getDataWithType } from "./context/utilities";
 import { MUTATION_KEYS, useMutation } from "../config/queryClient";
 import MultipleChoice from "./MultipleChoice";
 import TextInput from "./TextInput";
 import Slider from "./Slide";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import QuestionTopBar from "./QuestionTopBar";
 
 function Create() {
+
   const { data } = useAppData();
   const { mutate: postAppData } = useMutation(MUTATION_KEYS.POST_APP_DATA);
+  const { mutate: patchAppData } = useMutation(MUTATION_KEYS.PATCH_APP_DATA);
+  const [currentQuestionIndex, setCurrentQuestionIndex] = React.useState(0);
+  const [questionList, setQuestionList] = useState([]);
+  const [currentQuestionId, setCurrentQuestionId] = useState(null);
+  const [currentQuestionData, setCurrentQuestionData] = useState(null);
+  const questionListData = getDataWithType(data, QUESTION_LIST_TYPE)?.get(0)
```

Comment:
Hey Kim! Yes, indeed questionListData is merely just the constant expression of the whole questionList data structure in the database that is used twice in the code (for the actual questionList and its database id), but since it's only used twice, I can replace these two instances by the actual expression and remove the constant value.